ui/playwright.config.js

```diff
@@ -38,7 +41,7 @@ module.exports = defineConfig({
       mode: 'on-first-retry',
     },
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: 'on-first-retry',
```

Comment:
curious as the test suite grows if we will have to consider only keeping traces if the fail for ci perfermance. 

 This option keeps the trace on retry even if it succeeds/fails. Is that correct?

Reply:
curious thoughts on retain-on-failure option. As if we implemented 1 retry to reduce noise of flaky tests, we won't have to update this again. let me know your thoughts. Either way im good as along as there is awerness. thanks 🚀